### ISSUE TITLE
[Event Request] CU1012 "Job Jnl.-Post Line" - procedure Code()

### ISSUE THREAD

**Issue by CyrilleJsmt** [microsoft/ALAppExtensions#11360]
```
------- ADD PARAMETER : NextEntryNo ---------

    [IntegrationEvent(false, false)]
    local procedure OnAfterRunCode(var JobJournalLine: Record "Job Journal Line"; var JobLedgEntryNo: Integer; var JobRegister: Record "Job Register"; var NextEntryNo: Integer)
    begin
    end;

```
@AlexanderYakunin This fix is urgent, our client is stuck on validating his purchase invoices. Can we have this fix in CU5 (Update 17.5) please ?

**Comment by AlexanderYakunin** [comment]
Please let us know country code of your customer and is this cloud or on prem?

**Comment by CyrilleJsmt** [comment]
Country code : FR and our client is on prem.

**Comment by AlexanderYakunin** [comment]
Unfortunately, it is late to add any fix to 17.5, as deployment already started. We can add this fix to 17.x immediately, and we can hotfix 17.5 using ICM ticket just after release. The problem is how you can get it for your on prem customer? Do you know?

**Comment by CyrilleJsmt** [comment]
Currently with VSCode, we are using the Base application version "Microsoft_Base Application_17.4.21491.21531.app". Do you think you can update your app before version 17.6 ?

**Comment by AlexanderYakunin** [comment]
Absolutely, it will be part of 17.6.

**Comment by CyrilleJsmt** [comment]
Can we have the fix before the release of 17.6 in early April ?

**Comment by AlexanderYakunin** [comment]
You can make fix yourself, if you work with base app code and can compile app. If you want to get the fix earlier, you need to check with technical support if this is possible and how to do that and create hotfix request.
